cmd/kubelet/app/server.go

```diff
@@ -643,7 +643,18 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies, featureGate f
 		}
 
 		var reservedSystemCPUs cpuset.CPUSet
+		var staticCPUs cpuset.CPUSet
 		var errParse error
+
+		if s.StaticCPUs != "" {
+			staticCPUs, errParse = cpuset.Parse(s.StaticCPUs)
+			if errParse != nil {
```

Comment:
it's probably worth adding some error checking here to validate the cpu range falls within a range actually available on the host, and that the range doesn't overlap with something like "reserved-cpu" or any other conflicting options.